snowfakery/standard_plugins/Salesforce.py

```diff
@@ -114,17 +109,23 @@ def compose_query(self, context_name, **kwargs):
 
     @staticmethod
     def _get_sf_clients(orgname):
+
         try:
+            from cumulusci.cli.runtime import CliRuntime
```

Comment:
I thought you removed the dependency on CumulusCI? Is it present in certain contexts?

Reply:
Removed the hard (build-time, load-time) dependency. It still depends on CumulusCI if the user uses particular features.